### ISSUE TITLE
Inline style attributes not in output on server side render

### ISSUE THREAD

**Issue by niklaswolf** [HCESrl/html-to-vue#3]
Hi @Rigo-m,
we see a strange behaviour in shopware-pwa when using your library. I'll link to the corresponding issue there: https://github.com/vuestorefront/shopware-pwa/issues/1303 

I'll add a short description here, too:
Using the `h` function directly works! SSR Html contains the inline styles.
```
render(h, context) {
    const config = {...}
    
    const element = h("h1",{style: "text-align: center; color: #fff", domProps: {
        innerHTML: 'HElloooooo'
      }});
    return element;
  },
```

Using your library, inline styles are not in the HTML
```
render(h, context) {
    const config = {...}

    const rendered = renderHtml("<h1 style='text-align: center; color: #fff'>HElloooooo</h1>", config, h, context)
    return rendered;
  },
```

What's really odd is that the style attribute gets applied on the client side though, only the server rendered output is missing the style attributes in the outputted html when using you library.
Do you have an idea why this is happening?

Shopware-PWA uses Nuxt ^2.14.7 at the moment.

**Comment by Rigo-m** [comment]
I'll take a look at it ASAP

**Comment by Rigo-m** [comment]
@niklaswolf fixed in version 1.3.0
